port/unix/omrsysinfo.c

```diff
@@ -358,10 +358,22 @@ struct {
  * An example:
  *  7:cpuacct,cpu:/mycgroup
  */
-#define PROC_PID_CGROUP_ENTRY_FORMAT "%d:%[^:]:%s"
+#define PROC_PID_CGROUPV1_ENTRY_FORMAT "%d:%[^:]:%s"
 #define PROC_PID_CGROUP_SYSTEMD_ENTRY_FORMAT "%d::%s"
+/* For cgroup v2, the form is:
+ *  0::<cgroup name>
+ */
+#define PROC_PID_CGROUPV2_ENTRY_FORMAT "0::%s"
 
 #define SINGLE_CGROUP_METRIC 1
+#define MAX_64BIT_INT_LENGTH 20
+
+/* Used for PPG_sysinfoControlFlags global, set bits indicate cgroup v1 availability, cgroup v2 availability,
+ * and process running in container, respectively
```

Comment:
Each sentence should end with a period.

Please limit the width of lines; suggest a line break before 'cgroup v2' in this comment. Others comments should be wrapped likewise.